Kitodo/src/main/java/org/kitodo/production/helper/SearchResultGeneration.java

```diff
@@ -67,10 +69,25 @@ public HSSFWorkbook getResult() {
 
     private List<ProcessDTO> getResultsWithFilter() {
         List<ProcessDTO> processDTOS = new ArrayList<>();
+        int elasticsearchLimit = 9999;
```

Comment:
Just as a question: I mean we set the ElasticSearch limit in config.properties. Shouldn't you then have to use the value from the config.properties here? So, if someone has set a different value, does it still work as expected?

Reply:
It's a different elasticsearch.limit. It's the 10'000 limit which comes with elasticsearch. See also #4277 
So it's not configurable.

Reply:
OK